x/plans/keeper/migrations.go

```diff
@@ -91,3 +91,9 @@ func (m Migrator) Migrate4to5(ctx sdk.Context) error {
 func (m Migrator) Migrate5to6(ctx sdk.Context) error {
 	return m.keeper.plansFS.MigrateVersionFrom(ctx, 3)
 }
+
+// Migrate6to7 implements store migration from v6 to v7:
+// -- trigger fixation migration (v4->v5), initialize IsLatest field
+func (m Migrator) Migrate6to7(ctx sdk.Context) error {
+	return m.keeper.plansFS.MigrateVersionFrom(ctx, 4)
```

Comment:
why do we use the migrateversionfrom method instead of the general one?

Reply:
git-blame suggests you introduces the function MigrateVersionFrom() and its use in `x/plans/keeper/migrations.go:Migrate5to6() - which I used as base. I'm also curious why indeed?